modules/test/playwright/tests/portal-web/pageIteratorTaglib.spec.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+
+const test = mergeTests(loginTest());
+
+test(
+	'Check aria-label is being translated',
+	{tag: '@LPD-38101'},
+	async ({page}) => {
+		await test.step('Use search and change page language to check translation', async () => {
+			await page.goto('/es');
+
+			const searchBar = page.getByPlaceholder('Búsqueda…');
+
+			await searchBar.waitFor({state: 'visible'});
+
+			await searchBar.fill('png');
+
+			await searchBar.press('Enter');
```

Comment:
The three tests seem very similar. For performance, could we merge them into one? Each can be one step, after initial setup step.

Reply:
Sounds good!